src/components/app.tsx

```diff
@@ -9,10 +9,13 @@ import Onboarding from 'src/components/onboarding';
 import { SiteContentTabs } from 'src/components/site-content-tabs';
 import TopBar from 'src/components/top-bar';
 import UserSettings from 'src/components/user-settings';
+import WhatsNewModal from 'src/components/whats-new-modal';
```

Comment:
Should we keep component and hooks it in `modules/whats-new`?

Reply:
Sounds good to me. 👍🏼 I have made the change in https://github.com/Automattic/studio/pull/1110/commits/66758d9b03fc2a09bb1f2e9bdc4a17f5a4d8e6f5.